addons/website/views/snippets/snippets.xml

```diff
@@ -602,9 +602,27 @@
                        data-customize-website-variable="'slideout_shadow'"/>
         </we-select>
         <we-checkbox string="Copyright"
+                data-name="footer_copyright_opt"
                 data-customize-website-views="website.footer_no_copyright|"
                 data-no-preview="true"
                 data-reload="/"/>
+        <we-select string="Language Selector" data-dependencies="footer_copyright_opt" data-reload="/">
+            <we-button string="Inline"
+                       data-customize-website-views="website.footer_language_selector_inline"/>
+            <we-button string="Dropdown"
+                       data-customize-website-views="portal.footer_language_selector_dropdown"/>
+            <we-button string="None"
+                       data-name="language_selector_none_opt"
+                       data-customize-website-views=""/>
+        </we-select>
+        <we-select string="⌙ Label" data-dependencies="footer_copyright_opt" data-reload="/">
+            <we-button string="Text"
+                       data-customize-website-views=""/>
+            <we-button string="Flag"
+                       data-customize-website-views="website.footer_language_selector_flag, website.footer_language_selector_no_text"/>
+            <we-button string="Flag and Text"
+                       data-customize-website-views="website.footer_language_selector_flag"/>
+        </we-select>
```

Comment:
Appears even if "None" option selected... have you tested? :/
Also it actually was confusing to have all of this in the footer section instead of the copyright one...